modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/Source.java

```diff
@@ -27,37 +27,37 @@
  */
 public class Source {
 
-	private String LocURI;
-	private String LocName;
-
-	public String getLocURI() {
-		return LocURI;
-	}
-
-	public void setLocURI(String locURI) {
-		LocURI = locURI;
-	}
-
-	public String getLocName() {
-		return LocName;
-	}
-
-	public void setLocName(String locName) {
-		LocName = locName;
-	}
-
-	public void buildSourceElement(Document doc, Element rootElement) {
-		Element target = doc.createElement(Constants.SOURCE);
-		rootElement.appendChild(target);
-		if (getLocURI() != null) {
-			Element locURI = doc.createElement(Constants.LOC_URI);
-			locURI.appendChild(doc.createTextNode(getLocURI()));
-			target.appendChild(locURI);
-		}
-		if (getLocName() != null) {
-			Element locName = doc.createElement(Constants.LOC_NAME);
-			locName.appendChild(doc.createTextNode(getLocName()));
-			target.appendChild(locName);
-		}
-	}
+    private String LocURI;
+    private String LocName;
```

Comment:
Same goes here. Please follow the variable naming standards. First letter can not be capital.